websites/B/BeatLeader/presence.ts

```diff
@@ -1,10 +1,21 @@
 const presence = new Presence({
 		clientId: "939893188448104468",
 	}),
-	browsingTimestamp = Math.floor(Date.now() / 1000);
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	date = new Date(),
+	random = Math.floor(Math.random() * 100),
+	logoArr = [
+		"",
+		"https://i.imgur.com/dnRaNL0.gif", //Regular
+		"https://i.imgur.com/QdYNp1g.png", //BoooLeader
+		"https://i.imgur.com/cFuQ8gZ.png", //SpookLeader
+		"https://i.imgur.com/9HSrF7B.png", //Poodleader
+		"https://i.imgur.com/B6539OI.png", //GladdePaling
+		"https://i.imgur.com/bO3NiFN.gif", //EarthDay
+		"https://i.imgur.com/Kf8bXNc.gif", //Christmas
+	];
```

Comment:
Not sure if it's the smartest thing to use an array here, since I think you always pass a hard-coded index when requesting from it.

Reply:
> Not sure if it's the smartest thing to use an array here, since I think you always pass a hard-coded index when requesting from it.

It's also used by this setting
![opera_hi91nrtgB5](https://github.com/PreMiD/Presences/assets/44418429/bbd2bc24-0ec6-432d-9e82-4be9e8f77a12)